### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by lbialy** [okumin/akka-persistence-sql-async#17]
Hey @okumin, I've been trying to use this extension along with [quill](http://getquill.io) and since I'm forced to use Mauricio's `postgresql-async` @ 0.2.20 (due to quill dependency) and `akka-persistence-sql-async` uses `scalikejdbc-async` @ 0.5.5 which depends on `postgresql-async` @ 0.2.16 I get: 

```
java.lang.NoSuchMethodError: com.github.mauricio.async.db.Configuration$.$lessinit$greater$default$6()Ljava/nio/charset/Charset;
        at scalikejdbc.async.AsyncConnectionPool.<init>(AsyncConnectionPool.scala:33)
        at scalikejdbc.async.internal.postgresql.PostgreSQLConnectionPoolImpl.<init>(PostgreSQLConnectionPoolImpl.scala:37)
        at scalikejdbc.async.AsyncConnectionPoolFactory$.apply(AsyncConnectionPoolFactory.scala:43)
        at scalikejdbc.async.AsyncConnectionPool$.add(AsyncConnectionPool.scala:88)
        at akka.persistence.common.ScalikeJDBCExtension.<init>(ScalikeJDBCExtension.scala:20)
        at akka.persistence.common.ScalikeJDBCExtension$.createExtension(ScalikeJDBCExtension.scala:8)
        at akka.persistence.common.ScalikeJDBCExtension$.createExtension(ScalikeJDBCExtension.scala:6)
        at akka.actor.ActorSystemImpl.registerExtension(ActorSystem.scala:737)
        at akka.actor.ExtensionId$class.apply(Extension.scala:79)
        at akka.persistence.common.ScalikeJDBCExtension$.apply(ScalikeJDBCExtension.scala:6)
```

Is it possible for you to try and update this library to use newer `scalikejdbc-async` from 0.6.x branch which does support latest `postgresql-async`? I'll try to work on update on my own and send a PR, but I get some weird errors during project import in Idea and I'll have to set up databases for tests.

**Comment by lbialy** [comment]
Yep, this fixed conflicts with `scalikejdbc-async` and newer `postgresql-async´ required by Quill. Everything seems to work properly in my application.

**Comment by okumin** [comment]
I released 0.4.0, thanks!
https://github.com/okumin/akka-persistence-sql-async/releases/tag/v0.4.0
